lib/broccoli/fastboot-config.js

```diff
@@ -104,7 +104,9 @@ FastBootConfig.prototype.readAssetManifest = function() {
     var assetMap = JSON.parse(fs.readFileSync(assetMapPath));
     return assetMap;
   } catch (e) {
-    // No asset map was found, proceed as usual
+    if (assetMapPath) {
```

Comment:
In other words, I'm in favor of emitting a warning or error if we can't parse the file, or if we would expect the file to exist and it doesn't, but I am not in favor of logging a warning always in development mode if fingerprinting is turned off (which it is by default).